app/models/label/LabelTable.scala

```diff
@@ -785,6 +815,21 @@ object LabelTable {
     )
   }
 
+  /**
+    * Returns a LabelMetadataWithValidation object that has the label properties, tags, and includes 
```

Comment:
For [Scaladoc](https://docs.scala-lang.org/style/scaladoc.html) comments, we want to start with a short one-line summary of the method. If more detail is required, then a larger paragraph can be written on subsequent lines. I think this sentence could be made more concise/shorter and fit on one line.